subprojects/language-groovy/src/main/java/org/gradle/api/tasks/compile/GroovyCompile.java

```diff
@@ -116,20 +116,20 @@ public FileCollection call() {
     @CompileClasspath
     @Incremental
     public FileCollection getClasspath() {
-        // Note that this is an approximation and must be fixed before de-incubating getCompilerPluginClasspath()
+        // Note that this is an approximation and must be fixed before de-incubating getAstTransformationClasspath()
```

Comment:
Can you clarify the comment? I don't remember what is an approximation. Is `@CompileClasspath` an approximation? If yes, then I guess the comment should go there.

Reply:
Better now? https://github.com/gradle/gradle/pull/9794/commits/43eeb4c655a91964d067872bbc008c11ac482f9c